lms/djangoapps/teams/views.py

```diff
@@ -313,8 +313,8 @@ def get(self, request):
             )
         else:
             return Response({
-                'developer_message': "unsupported order_by value {}".format(order_by_input),
-                'user_message': _(u"The ordering {} is not supported").format(order_by_input),
+                'developer_message': "unsupported order_by value {ordering}".format(ordering=order_by_input),
+                'user_message': _(u"The ordering {ordering} is not supported").format(ordering=order_by_input),
```

Comment:
Out of context, I have absolutely no clue what this means. Please add a translator's comment. See http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#editing-source-files for syntax notes on how to do this in every file type.